web/src/features/hfiCalculator/components/WeeklyViewTable.tsx

```diff
@@ -12,19 +12,20 @@ import BaseStationAttributeCells from 'features/hfiCalculator/components/BaseSta
 import GrassCureCell from 'features/hfiCalculator/components/GrassCureCell'
 import { isGrassFuelType } from 'features/hfiCalculator/validation'
 import { BACKGROUND_COLOR, fireTableStyles } from 'app/theme'
-import { isEmpty, union } from 'lodash'
+import { isEmpty } from 'lodash'
 import { StationDaily } from 'api/hfiCalculatorAPI'
 import { getDailiesByStationCode, getZoneFromAreaName } from 'features/hfiCalculator/util'
 import StickyCell from 'components/StickyCell'
 import FireCentreCell from 'features/hfiCalculator/components/FireCentreCell'
-import { selectHFIPrepDays } from 'app/rootReducer'
+import { selectHFICalculatorState } from 'app/rootReducer'
 import { useSelector } from 'react-redux'
 
 export interface Props {
+  testId?: string
   fireCentres: Record<string, FireCentre>
   dailies: StationDaily[]
   currentDay: string
-  testId?: string
+  setSelected: (selected: number[]) => void
```

Comment:
being pretty picky, but ideally this variable name would be expanded to specify what has been selected. i.e., I had to scroll further down in the code to know that it's referring to which stations have been selected. Looking at the name alone, it could've meant the date selected, the prep day selected, (soon - the fire centre selected), etc. ...

Reply:
`explicit` > `implicit`, I agree

Reply:
https://www.python.org/dev/peps/pep-0020/